Fine tune doc for eventing (#4145)
kyma-project/kyma: docs/event-bus/02-01-event-bus.md

1. A user creates a lambda or a service that an Event coming from an external solution triggers. 

>     >**NOTE**: For a service, the user must create a Kyma Subscription resource manually. If the user is creating a lambda in Kyma Console UI then it is created automatically else the user needs to create the Kyma Subscription manually.

Reviewer: ```suggestion
    >**NOTE**: When creating a service, the user must create a Kyma Subscription resource manually. If the user uses Kyma Console UI to create a lambda, the Subscription resource is created automatically. In any other case, Kyma Subscription must be created manually.
```

Author: Changed
